File: Blackjack21-JAVA/Blackjack21Library/src/Blackjack21/Game/Exceptions/PlayerNameEmptyException.java
package Blackjack21.Game.Exceptions;

/**
 * Throws an exception when the player's name is empty
 */
public class PlayerNameEmptyException extends Exception
{

    public PlayerNameEmptyException()
    {
        super("Player name cannot be empty");
    }

    public PlayerNameEmptyException(String message)
    {
        super(message);
    }

}

File: BlackJack-CSHARP/Blackjack21.Game/Game/Exceptions/PlayerNameEmptyException.cs
using System;

namespace Blackjack21.Game.Exceptions
{
    /// <summary>
    /// Throws an exception when the player's name is empty
    /// </summary>
    public class PlayerNameEmptyException : Exception
    {

        public PlayerNameEmptyException(string message="Player name cannot be empty"): base(message)
        {

        }

    }
}
